packages/rum-react/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@elastic/apm-rum-react",
+  "version": "0.0.0",
+  "description": "Elastic APM RUM React integration",
```

Comment:
Just to keep it consistent with our apm-rum,  We can rename it as `Elastic APM JavaScript agent for React Applications`?  WDYT?

Reply:
I've change it a little bit, I think we should include "Real User Monitoring" in some form in the description. We should fix apm-rum as well.